src/bin/drag_and_drop_listbox.rs

```diff
@@ -0,0 +1,193 @@
+// A drag-and-drop example. You can drag the list items using the icon on the left, and drop them
+// to other icons to move the list item. Originally listbox-dnd.c in GTK+ repository. Ported to
+// gtk-rs by Ömer Sinan Ağacan <omeragacan@gmail.com>.
+
+extern crate cairo;
+extern crate gdk;
+extern crate gio;
+extern crate gtk;
+
+use gdk::{Atom, DragAction, DragContext, ModifierType, Screen};
+use gio::prelude::*;
+use gtk::*;
```

Comment:
Don't do wildcard imports from `gtk`, if only from the `prelude`. Especially for an example it's otherwise not really clear where types come from :)